inputfiles/browser.webidl.xml

```diff
@@ -3560,6 +3560,8 @@
         <property name="onended" event-handler="ended" type="EventHandler"/>
         <property name="onerror" event-handler="error" type="EventHandler"/>
         <property name="onfocus" event-handler="focus" type="EventHandler"/>
+        <property name="focusin" event-handler="focusin" type="EventHandler"/>
```

Comment:
these files are autogrenerated and we do not manually change them.So we will need to add support for adding even handler maps to the generation script and add them to overridingTypes.json instead.

Reply:
@mhegazy thanks, I'll update the PR.

Reply:
Hi @mhegazy 

I have added following to addedTypes.json but it doesn't generate anything. Do I have to update somewhere else too? I suspect that it's the double quotes in DocumentEventMap properties. (ex: `"blur": FocusEvent;` ) but not sure.

```JavaScript
{
        "kind": "property",
        "interface": "DocumentEventMap",
        "name": "\"focusin\"",
        "type": "FocusEvent"
    },
    {
        "kind": "property",
        "interface": "DocumentEventMap",
        "name": "\"focusout\"",
        "type": "FocusEvent"
    }

```